$OpenBSD: patch-configure,v 1.1.1.1 2011/09/15 15:20:17 jasper Exp $

- Fixup after ${SUBST_CMD} mangled this file
- Fix path to qt library

--- configure.orig	Thu Mar 17 20:57:40 2011
+++ configure	Wed Sep  7 18:52:25 2011
@@ -2511,15 +2511,8 @@ echo timestamp > conftest.file
 # name.  Accept space and tab only in the latter.
 am_lf='
 '
-case `pwd` in
-  *[\\\"\#\$\&\'\`$am_lf]*)
-    as_fn_error $? "unsafe absolute working directory name" "$LINENO" 5 ;;
-esac
-case $srcdir in
-  *[\\\"\#\$\&\'\`$am_lf\ \	]*)
-    as_fn_error $? "unsafe srcdir value: \`$srcdir'" "$LINENO" 5 ;;
-esac
 
+
 # Do `set' in a subshell so we don't clobber the current shell's
 # arguments.  Must try -L first in case configure is actually a
 # symlink; some systems play weird games with the mod time of symlinks
@@ -5592,7 +5585,7 @@ $as_echo_n "checking for Qt headers... " >&6; }
 paths="$QTDIR/include /usr/local/qt/include /usr/include/qt /usr/include/qt3 \
   /usr/include /usr/X11R6/include/X11/qt /usr/X11R6/include/X11/qt \
   /usr/X11R6/include/qt /usr/X11R6/include /sw/include/qt \
-  /usr/X11R6/include/qt2 $prefix/include/qt3"
+  /usr/X11R6/include/qt2 $prefix/include/qt3 ${LOCALBASE}/include/X11/qt3/"
 for path in $paths; do
   if test -f "$path/qapplication.h"; then
     QT_INCLUDES=$path
